plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -91,6 +91,8 @@ func ClusterRoles() []rbac.ClusterRole {
 			Rules: []rbac.PolicyRule{
 				// TODO add future selfsubjectrulesreview, project request APIs, project listing APIs
 				rbac.NewRule("create").Groups(authorizationGroup).Resources("selfsubjectaccessreviews").RuleOrDie(),
+
+				rbac.NewRule(Read...).Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
```

Comment:
I'm still not settled with adding this to the "system:basic-user" role. It made sense to me that the one thing that this role allows someone to do is to figure out what he's allowed to do.

Adding in "storageclasses" feels like the beginning of an accumulation of similar things ambient to the cluster that could sit together in a grab bag. I just don't know what those other things are today. Hence, I have a hard time coming up with the name for a role that would gather these things. Maybe "system:cluster-config," or "system:cluster-facilities?"